core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -88,9 +88,91 @@ beforeAll(() => {
+       browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+       browser.sleep(45000);
+       browser.refresh();
+       browser.driver.switchTo().activeElement();
+       jazzServices_po.getRefresh().click();
+       browser.sleep(30000);
+  });
+  it('Verify API Prod Service', () => {
+      browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+      jazzServices_po.getProdName().click();
+  });   
+  it('Verify API Asset Service', () => {
+      browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+      browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+      browser.driver.switchTo().activeElement();
+      browser.sleep(15000);
+      jazzServices_po.getRefresh().click();
+      browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+      jazzServices_po.getRefresh().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getAsset()), timeOutHigh);
+      jazzServices_po.getAsset().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getAssetHeader()), timeOutHigh);
+      browser.sleep(4000);
+      browser.wait(EC.elementToBeClickable(jazzServices_po.getServiceFromAsset()), timeOutHigh);
+      jazzServices_po.getServiceFromAsset().click();
+  });
+  it('Verify lamda function Page Title', () => {
+      browser.wait(EC.visibilityOf(jazzServices_po.getAPIServiceName()), timeOutHigh);
+      browser.wait(EC.elementToBeClickable(jazzServices_po.getAPIServiceName()), timeOutHigh);
+      jazzServices_po.getLamdaName().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+      browser.sleep(45000);
+      browser.refresh();
+      browser.driver.switchTo().activeElement();
+      jazzServices_po.getRefresh().click();
+      browser.sleep(40000);
+      browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+      jazzServices_po.getProdName().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+      browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+      browser.driver.switchTo().activeElement();
+      browser.sleep(15000);
+      jazzServices_po.getRefresh().click();
+      browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+      jazzServices_po.getRefresh().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getAsset()), timeOutHigh);
+      jazzServices_po.getAsset().click();
+      browser.wait(EC.visibilityOf(jazzServices_po.getAssetHeader()), timeOutHigh);
+      browser.sleep(4000);
+      browser.wait(EC.elementToBeClickable(jazzServices_po.getServiceFromAsset()), timeOutHigh);
+      jazzServices_po.getServiceFromAsset().click();
+  });
+it('Verify Website Page Title', () => {
+      browser.wait(EC.visibilityOf(jazzServices_po.getAPIServiceName()), timeOutHigh);
```

Comment:
just curious why all the code is combined for website and lambda but for api it is broken down?

Reply:
Will make it similar

Reply:
Actually it will not affect the functionalities,but I changed and made it similar